docker-compose.yml

```diff
@@ -46,6 +46,13 @@ services:
     image: celeryapp
     restart: unless-stopped
     build: dc_config/images/celery
+    cap_add:
```

Comment:
SYS_ADMIN capability opens up security holes. Is this a requirement to be able to use s3fs_fuse in the container?

Reply:
Ya I think we need the SYS_ADMIN capability.
here is an error which I had followed through to mount s3fs.
https://github.com/s3fs-fuse/s3fs-fuse/issues/647